### ISSUE TITLE
environment.yaml

### ISSUE THREAD

**Issue by ZBoIsHere** [ZBoIsHere/ZBoIsHere#12]
name: /root/occformer/nerf_zju
channels:
  - conda-forge
  - pytorch
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=2_gnu
  - blas=1.0=mkl
  - brotli=1.0.9=h166bdaf_9
  - brotli-bin=1.0.9=h166bdaf_9
  - brotli-python=1.0.9=py38h6a678d5_7
  - bzip2=1.0.8=h7f98852_4
  - ca-certificates=2024.3.11=h06a4308_0
  - certifi=2024.2.2=py38h06a4308_0
  - charset-normalizer=3.1.0=pyhd8ed1ab_0
  - cudatoolkit=11.3.1=h2bc3f7f_2
  - docopt=0.6.2=py38h06a4308_1
  - ffmpeg=4.3=hf484d3e_0
  - filelock=3.12.2=pyhd8ed1ab_0
  - freetype=2.12.1=hca18f0e_1
  - gmp=6.2.1=h58526e2_0
  - gmpy2=2.1.2=py38h793c122_1
  - gnutls=3.6.13=h85f3911_1
  - icu=72.1=hcb278e6_0
  - idna=3.4=pyhd8ed1ab_0
  - intel-openmp=2023.1.0=hdb19cb5_46305
  - jinja2=3.1.2=pyhd8ed1ab_1
  - jpeg=9e=h0b41bf4_3
  - lame=3.100=h166bdaf_1003
  - lcms2=2.15=hfd0df8a_0
  - ld_impl_linux-64=2.40=h41732ed_0
  - lerc=4.0.0=h27087fc_0
  - libbrotlicommon=1.0.9=h166bdaf_9
  - libbrotlidec=1.0.9=h166bdaf_9
  - libbrotlienc=1.0.9=h166bdaf_9
  - libdeflate=1.17=h0b41bf4_0
  - libffi=3.4.2=h7f98852_5
  - libgcc-ng=13.1.0=he5830b7_0
  - libgomp=13.1.0=he5830b7_0
  - libhwloc=2.9.1=nocuda_h7313eea_6
  - libiconv=1.17=h166bdaf_0
  - libnsl=2.0.0=h7f98852_0
  - libpng=1.6.39=h753d276_0
  - libsqlite=3.42.0=h2797004_0
  - libstdcxx-ng=13.1.0=hfd8a6a1_0
  - libtiff=4.5.0=h6adf6a1_2
  - libuuid=2.38.1=h0b41bf4_0
  - libwebp-base=1.3.0=h0b41bf4_0
  - libxcb=1.13=h7f98852_1004
  - libxml2=2.11.4=h0d562d8_0
  - libzlib=1.2.13=hd590300_5
  - markupsafe=2.1.3=py38h01eb140_0
  - mkl=2023.1.0=h6d00ec8_46342
  - mkl-service=2.4.0=py38h5eee18b_1
  - mkl_fft=1.3.6=py38h417a72b_1
  - mkl_random=1.2.2=py38h417a72b_1
  - mpc=1.3.1=hfe3b2da_0
  - mpfr=4.2.0=hb012696_0
  - mpmath=1.3.0=pyhd8ed1ab_0
  - ncurses=6.4=hcb278e6_0
  - nettle=3.6=he412f7d_0
  - networkx=3.1=pyhd8ed1ab_0
  - numpy-base=1.24.3=py38h060ed82_1
  - openh264=2.1.1=h4ff587b_0
  - openjpeg=2.5.0=hfec8fc6_2
  - openssl=3.1.1=hd590300_1
  - pip=23.1.2=pyhd8ed1ab_0
  - pipreqs=0.4.13=pyhd8ed1ab_0
  - pthread-stubs=0.4=h36c2ea0_1001
  - pysocks=1.7.1=pyha2e5f31_6
  - python=3.8.17=he550d4f_0_cpython
  - python_abi=3.8=3_cp38
  - pytorch-mutex=1.0=cpu
  - readline=8.2=h8228510_1
  - sympy=1.12=pypyh9d50eac_103
  - tbb=2021.9.0=hf52228f_0
  - tk=8.6.12=h27826a3_0
  - typing_extensions=4.6.3=pyha770c72_0
  - wheel=0.40.0=pyhd8ed1ab_0
  - xorg-libxau=1.0.11=hd590300_0
  - xorg-libxdmcp=1.1.3=h7f98852_0
  - xz=5.2.6=h166bdaf_0
  - yarg=0.1.9=py_1
  - zlib=1.2.13=hd590300_5
  - zstd=1.5.2=h3eb15da_6
  - pip:
      - absl-py==1.4.0
      - addict==2.4.0
      - aenum==3.1.15
      - aiohttp==3.8.5
      - aiosignal==1.3.1
      - aliyun-python-sdk-core==2.14.0
      - aliyun-python-sdk-kms==2.16.2
      - ansi2html==1.8.0
      - antlr4-python3-runtime==4.9.3
      - anykeystore==0.2
      - apex==0.9.10.dev0
      - array-record==0.4.0
      - asttokens==2.2.1
      - astunparse==1.6.3
      - async-timeout==4.0.3
      - attrs==23.1.0
      - backcall==0.2.0
      - beautifulsoup4==4.12.2
      - cachetools==5.3.1
      - cffi==1.16.0
      - click==8.1.3
      - cloudpickle==2.2.1
      - cmake==3.27.4.1
      - colorama==0.4.6
      - coloredlogs==15.0.1
      - comm==0.1.3
      - configargparse==1.5.5
      - contourpy==1.1.0
      - crcmod==1.7
      - cryptacular==1.6.2
      - cryptography==41.0.4
      - cupy-cuda11x==12.2.0
      - cycler==0.11.0
      - cython==3.0.0
      - dacite==1.8.1
      - dash==2.11.0
      - dash-core-components==2.0.0
      - dash-html-components==2.0.0
      - dash-table==5.0.0
      - dask==2023.3.1
      - dataclass-array==1.4.1
      - debugpy==1.6.7
      - decorator==5.1.1
      - defusedxml==0.7.1
      - dinov2==0.0.1
      - dm-tree==0.1.8
      - einops==0.6.1
      - einsum==0.3.0
      - etils==1.3.0
      - executing==1.2.0
      - fake-factory==9999.9.9
      - fastjsonschema==2.17.1
      - fastrlock==0.8.1
      - flask==2.2.5
      - flatbuffers==23.5.26
      - fonttools==4.40.0
      - freetype-py==2.4.0
      - frozenlist==1.4.0
      - fsspec==2023.6.0
      - ftfy==6.1.1
      - fvcore==0.1.5.post20221221
      - gast==0.4.0
      - gdown==4.7.1
      - google-auth==2.16.2
      - google-auth-oauthlib==0.4.6
      - google-pasta==0.2.0
      - googleapis-common-protos==1.60.0
      - greenlet==2.0.2
      - grpcio==1.56.2
      - h5py==3.9.0
      - huggingface-hub==0.16.4
      - humanfriendly==10.0
      - hupper==1.12
      - icecream==2.1.3
      - imageio==2.31.1
      - imgviz==1.5.0
      - immutabledict==2.2.0
      - importlib-metadata==6.7.0
      - importlib-resources==5.12.0
      - iopath==0.1.10
      - ipycanvas==0.13.1
      - ipyevents==2.0.1
      - ipykernel==6.23.3
      - ipython==8.12.2
      - ipywidgets==8.0.6
      - itsdangerous==2.1.2
      - jedi==0.18.2
      - jmespath==0.10.0
      - joblib==1.3.0
      - jsonschema==4.17.3
      - jupyter-client==7.4.9
      - jupyter-core==5.3.1
      - jupyterlab-widgets==3.0.7
      - keras==2.11.0
      - kitti360scripts==1.0.0
      - kiwisolver==1.4.4
      - kornia==0.6.12
      - lark==1.1.7
      - lazy-loader==0.3
      - libclang==16.0.6
      - lietorch==0.2
      - lightning-utilities==0.9.0
      - lit==16.0.6
      - locket==1.0.0
      - lpips==0.1.4
      - markdown==3.4.4
      - markdown-it-py==3.0.0
      - matplotlib==3.6.1
      - matplotlib-inline==0.1.6
      - mdurl==0.1.2
      - meshio==5.3.4
      - mmcls==0.25.0
      - mmcv==2.1.0
      - mmdeploy==1.3.0
      - mmengine==0.9.0
      - model-index==0.1.11
      - multidict==6.0.4
      - multiprocess==0.70.15
      - natsort==8.1.0
      - nbformat==5.7.0
      - nest-asyncio==1.5.6
      - ninja==1.11.1
      - numpy==1.22.4
      - nvdiffrast==0.3.1
      - nvidia-cublas-cu11==11.10.3.66
      - nvidia-cuda-cupti-cu11==11.7.101
      - nvidia-cuda-nvrtc-cu11==11.7.99
      - nvidia-cuda-runtime-cu11==11.7.99
      - nvidia-cudnn-cu11==8.5.0.96
      - nvidia-cufft-cu11==10.9.0.58
      - nvidia-curand-cu11==10.2.10.91
      - nvidia-cusolver-cu11==11.4.0.1
      - nvidia-cusparse-cu11==11.7.4.91
      - nvidia-nccl-cu11==2.14.3
      - nvidia-nvtx-cu11==11.7.91
      - oauthlib==3.2.2
      - omegaconf==2.3.0
      - onnx==1.13.1
      - onnxruntime==1.14.1
      - open3d==0.17.0
      - opencv-contrib-python==4.8.0.76
      - opencv-contrib-python-headless==4.7.0.72
      - opencv-python==4.8.0.76
      - opencv-python-headless==4.8.0.76
      - opendatalab==0.0.10
      - openexr==1.3.9
      - openxlab==0.0.28
      - opt-einsum==3.3.0
      - ordered-set==4.1.0
      - oss2==2.17.0
      - packaging==23.1
      - pandas==1.5.3
      - parso==0.8.3
      - partd==1.4.0
      - pastedeploy==3.0.1
      - pbkdf2==1.3
      - pexpect==4.8.0
      - pickleshare==0.7.5
      - pillow==9.2.0
      - pkgutil-resolve-name==1.3.10
      - plaster==1.1.2
      - plaster-pastedeploy==1.0.1
      - platformdirs==3.8.0
      - plotly==5.13.1
      - polyscope==1.3.3
      - portalocker==2.7.0
      - prettytable==3.9.0
      - promise==2.3
      - prompt-toolkit==3.0.38
      - protobuf==3.20.2
      - psutil==5.9.5
      - ptyprocess==0.7.0
      - pure-eval==0.2.2
      - pyarrow==10.0.0
      - pyasn1==0.5.0
      - pyasn1-modules==0.3.0
      - pybind11==2.10.4
      - pycocotools==2.0.7
      - pycparser==2.21
      - pycryptodome==3.19.0
      - pyglet==1.5.28
      - pygments==2.15.1
      - pyhocon==0.3.57
      - pymcubes==0.1.4
      - pymeshlab==2022.2.post4
      - pyopengl==3.1.0
      - pyopengl-accelerate==3.1.7
      - pyparsing==3.1.0
      - pyqt5==5.15.7
      - pyqt5-qt5==5.15.2
      - pyqt5-sip==12.12.1
      - pyquaternion==0.9.9
      - pyramid==2.0.1
      - pyramid-mailer==0.15.1
      - pyrender==0.1.45
      - pyrsistent==0.19.3
      - python-dateutil==2.8.2
      - python3-openid==3.2.0
      - pytorch-lightning==2.0.7
      - pytorch3d==0.3.0
      - pytz==2023.3
      - pywavelets==1.4.1
      - pyyaml==6.0
      - pyzmq==24.0.1
      - qimage2ndarray==1.10.0
      - regex==2023.10.3
      - repoze-sendmail==4.4.1
      - requests==2.28.2
      - requests-oauthlib==1.3.1
      - retrying==1.3.4
      - rich==13.4.2
      - rsa==4.9
      - safetensors==0.3.3
      - sampler==0.2.0
      - scikit-image==0.21.0
      - scikit-learn==1.2.2
      - scipy==1.7.2
      - seaborn==0.12.2
      - segment-anything==1.0
      - setuptools==60.2.0
      - shencoder==0.0.0
      - six==1.16.0
      - smooth-sampler==0.0.1
      - soupsieve==2.4.1
      - sqlalchemy==2.0.19
      - stack-data==0.6.2
      - supervision==0.6.0
      - tabulate==0.9.0
      - tenacity==8.2.2
      - tensorboard==2.11.2
      - tensorboard-data-server==0.6.1
      - tensorboard-plugin-wit==1.8.1
      - tensorflow-addons==0.21.0
      - tensorflow-datasets==4.9.0
      - tensorflow-estimator==2.11.0
      - tensorflow-graphics==2021.12.3
      - tensorflow-io-gcs-filesystem==0.33.0
      - tensorflow-metadata==1.13.0
      - tensorflow-probability==0.19.0
      - termcolor==1.1.0
      - terminaltables==3.1.10
      - threadpoolctl==3.1.0
      - tifffile==2023.4.12
      - timm==0.9.6
      - tinycudann==1.7
      - tinyobjloader==2.0.0rc7
      - tokenizers==0.14.1
      - toml==0.10.2
      - tomli==2.0.1
      - toolz==0.12.0
      - torch==1.11.0+cu113
      - torchaudio==0.11.0+cu113
      - torchmetrics==1.1.1
      - torchvision==0.12.0+cu113
      - tornado==6.3.2
      - tqdm==4.65.0
      - traitlets==5.9.0
      - transaction==3.1.0
      - transformers==4.35.0
      - translationstring==1.4
      - trimesh==3.22.5
      - triton==2.0.0
      - typeguard==2.13.3
      - tzdata==2023.3
      - urllib3==1.26.16
      - usd-core==22.5
      - velruse==1.1.1
      - venusian==3.0.0
      - visu3d==1.5.1
      - waymo-open-dataset-tf-2-11-0==1.6.0
      - wcwidth==0.2.6
      - webob==1.8.7
      - werkzeug==2.2.3
      - widgetsnbextension==4.0.7
      - wtforms==3.0.1
      - wtforms-recaptcha==0.3.2
      - xatlas==0.0.8
      - yacs==0.1.8
      - yapf==0.40.2
      - yarl==1.9.2
      - zipp==3.15.0
      - zope-deprecation==5.0
      - zope-interface==6.0
      - zope-sqlalchemy==3.0
prefix: /root/occformer/nerf_zju
